src/app/shared/services/project/project.service.ts

```diff
@@ -94,7 +94,7 @@ export interface Project {
     documents: DocumentModel[];
     gallery: string[];
     active: boolean;
-    organization: Organization[];
```

Comment:
This is not an array. Please, find out how to set template Typescript type checking for Angular inside templates (.html files). You would notice this on `{{ states.project.organization?.name }}` if you had this check.